doc/1-intro/quickstart.md

```diff
@@ -0,0 +1,230 @@
+# quick start guide
+
+first things first: thank you for taking the time to understand Furnace, the world's most comprehensive chiptune tracker. it's amazingly versatile, but it can also be intimidating, even for those already familiar with trackers. this quick start guide will get you on the road to making the chiptunes of your dreams! if you're a beginner, it will probably take about an hour <!-- estimate, test and revise it when done --> from start to finish.
```

Comment:
> first things first: thank you for taking the time to understand Furnace, the world's most comprehensive chiptune tracker.

I suggest you not start the guide this way. I don't know how to explain it, but it doesn't feel right for some reason...

Reply:
Which part, the "taking the time to understand Furnace" part, or the "world's most comprehensive chiptune tracker" part? I could easily get rid of both and just make it "welcome to the Furnace chiptune tracker!"

Reply:
The entire line.